cmd/cstor-volume-mgmt/controller/volume-controller/handler.go

```diff
@@ -118,8 +118,16 @@ func (c *CStorVolumeController) cStorVolumeEventHandler(
 		if err != nil {
 			return common.CVStatusError, err
 		}
+		//TODO: Should we need to make update call here itself or caller of the
+		//code will take care?
 
-		return common.CVStatusIgnore, nil
+		// IsZero returns true if the quantity is equal to zero.
+		if cStorVolumeGot.Status.Capacity.IsZero() {
```

Comment:
you can remove this 'if' check

Reply:
Means whatever it is there we need to override it?

Reply:
Removed `IsZero` check